e2e/js_image/js_image_layer.bzl

```diff
@@ -0,0 +1,247 @@
+"contains container helper functions for js_binary"
```

Comment:
TODO about promoting it to the rules_js public api? We don't want the long term state to be users copy-paste this code right?

Reply:
Right. we'll promote it to public api before 1.0

Reply:
Filed #304